README.md

```diff
@@ -70,6 +71,19 @@ end
 
 More details on https://hexdocs.pm/tower/Tower.html#module-manual-reporting.
 
+### Configuration
+
+`TowerSentry` supports the following configuration options:
+
+- `:dsn` (`t:String.t/0`) - The DSN for your Sentry project. Setting this option is mandatory. Learn more about DSNs in the [official Sentry documentation](https://docs.sentry.io/concepts/key-terms/dsn-explainer/).
+- `:environment_name` (`t:String.t/0` or `t:atom/0`) - The current environment name. Setting this option is mandatory. Learn more about environments in the [official Sentry documentation](https://docs.sentry.io/concepts/key-terms/environments/).
```

Comment:
Maybe this should be moved to another page? Github markdown obviously doesn't support ExDoc's syntax for auto-linking so the readme looks a little funny.

Reply:
Keeping it in the README for now it's fine.